ddl/column.go

```diff
@@ -51,36 +51,6 @@ import (
 	"go.uber.org/zap"
 )
 
-// adjustColumnInfoInAddColumn is used to set the correct position of column info when adding column.
```

Comment:
We need to comment that the new adding column would be moved to the end of tblInfo.Columns